bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrPlugin.kt

```diff
@@ -46,7 +46,8 @@ internal class AnrPlugin : Plugin {
      * Notifies bugsnag that an ANR has occurred, by generating an Error report and populating it
      * with details of the ANR. Intended for internal use only.
      */
-    private fun notifyAnrDetected() {
+    @Suppress("UNUSED_PARAMETER")
+    private fun notifyAnrDetected(info: Long, user_context: Long) {
```

Comment:
```suggestion
    private fun notifyAnrDetected(info: Long, userContext: Long) {
```

I believe this should resolve the static analysis violation from Detekt - Kotlin's naming convention is to use camel case.

Reply:
Updated to use camel case. Unfortunately, it still need the @Suppress to compile. It will be removed once all the parts are connected, though.